packages/node-http-handler/src/node-http-handler.ts

```diff
@@ -59,12 +68,14 @@ export class NodeHttpHandler implements HttpHandler {
   }
 
   private resolveDefaultConfig(options?: NodeHttpHandlerOptions | void): ResolvedNodeHttpHandlerConfig {
-    const { connectionTimeout, socketTimeout, httpAgent, httpsAgent } = options || {};
+    const { requestTimeout, connectionTimeout, socketTimeout, httpAgent, httpsAgent } = options || {};
     const keepAlive = true;
     const maxSockets = 50;
+
     return {
       connectionTimeout,
       socketTimeout,
+      requestTimeout: requestTimeout ?? connectionTimeout ?? socketTimeout ?? DEFAULT_REQUEST_TIMEOUT,
```

Comment:
this priority order should be mentioned in the deprecation message of the two fields